### ISSUE TITLE
Missing symbols set_memory_rw and set_memory_ro

### ISSUE THREAD

**Issue by ohEmily** [euspectre/kedr#27]
Hi there,

I'm running Arch with kernel 4.1.15. and kedr as pulled straight from this repo.

When I try to use kedr with a module (i.e. kedr start <modulename>), it fails:
```
/usr/bin/insmod /lib/modules/4.1.15-1-lts/extra/kedr.ko target_name=simple
insmod: ERROR: could not insert module /lib/modules/4.1.15-1-lts/extra/kedr.ko: Unknown symbol in module
Error: Failed to execute commands to start KEDR. Performing rollback.
``

Running dmesg reveals that the issue is that there's certain kernel symbols missing:
```
[  118.166144] kedr: Unknown symbol set_memory_rw (err 0)
[  118.166191] kedr: Unknown symbol set_memory_ro (err 0)
```

Looking around online, the issue is that these symbols are no longer exported in the kernel:
```
commit 6bbb614ec478961c7443086bdf7fd6784479c14a
Author: Daniel Borkmann <daniel@iogearbox.net>
Date:   Fri Feb 27 15:55:40 2015 +0100

    x86/mm: Unexport set_memory_ro() and set_memory_rw()
```

**Comment by euspectre** [comment]
Yes, I stumbled upon this issue as well. It will be needed to find the addresses of these functions via kallsyms. Kind of ugly, but should work.

It will try to provide the fix as soon as I have time.

**Comment by euspectre** [comment]
Should be fixed by commit 85c04d9. 
Please check if KEDR works OK on your system now.

**Comment by ohEmily** [comment]
It works! Thank you for getting to this so quickly.
